src/gui/folderman.cpp

```diff
@@ -849,7 +849,7 @@ Folder *FolderMan::folderForPath(const QString &path)
     foreach(Folder* folder, this->map().values()) {
         const QString folderPath = folder->cleanPath()+QLatin1Char('/');
 
-        if(absolutePath.startsWith(folderPath)) {
+        if(absolutePath.startsWith(folderPath, Qt::CaseInsensitive)) {
```

Comment:
The file system has the case-sensitive option when you format, but you're right, the default is incensitive while I thought that it was sensitive 😮
So please ignore my request.